ksql-engine/src/test/java/io/confluent/ksql/physical/PhysicalPlanBuilderTest.java

```diff
@@ -116,15 +116,15 @@ private PhysicalPlanBuilder buildPhysicalPlanBuilder(Map<String, Object> overrid
     configMap.put("auto.offset.reset", "earliest");
     ksqlConfig = new KsqlConfig(configMap);
     return new PhysicalPlanBuilder(streamsBuilder,
-        ksqlConfig,
-        new FakeKafkaTopicClient(),
-        functionRegistry,
-        overrideProperties,
-        false,
-        metaStore,
-                                                  new MockSchemaRegistryClient(),
-                                                  new QueryIdGenerator(),
-        testKafkaStreamsBuilder
+                                   ksqlConfig,
```

Comment:
I don't think these changes match the confluent-google style template. They looked correct before. Make sure you are using the correct ide settings otherwise we end up with back and forth on whitespace changes

Reply:
Reformatted with our code style.